feeder/parsers/PL.py

```diff
@@ -0,0 +1,69 @@
+
+from bs4 import BeautifulSoup
+import requests
+import arrow
+
+COUNTRY_CODE = 'PL'
+
+def fetchValue(params):
+
+    now = arrow.utcnow()
+    end = now.replace(hours=+2)
+    start = now.replace(hours=-22)
+    periodEnd = str(end.format('YYYYMMDDHH00'))
+    periodStart = str(start.format('YYYYMMDDHH00'))
+
+
+
+    parameters = '&psrType=' + params + '&documentType=A75&processType=A16&in_Domain=10YPL-AREA-----S&periodStart=' + periodStart + '&periodEnd=' + periodEnd
+    link = 'https://transparency.entsoe.eu/api?securityToken=7466690c-c66a-4a00-8e21-2cb7d538f380' + parameters
+
+    content = requests.get(link)
+    soup = BeautifulSoup(content.text, "html.parser")
+
+    quantity = []
+
+    item = soup.find_all('point')
+    for pos in item:
+        p = pos.find_all('quantity')
```

Comment:
if you want the last you can just write 
`last = soup.find_all('point')[-1].find_all('quantity')[-1]`